manoseimas/mps_v2/static/components/switcher_component.jsx

```diff
@@ -12,8 +14,100 @@ var Switcher = React.createClass({
+      self.setState(function(previousState, currentProps) {
+        var new_subtabs = {};
+        var old_subtabs = previousState.active_subtabs;
+        Object.keys(old_subtabs).map( function(key) {
+          key === tab_name ? val = subtab_name : val = old_subtabs[key];
+          new_subtabs[key] = val;
+        });
+        return {active_subtabs: new_subtabs};
+      });
+  },
+
+  getSubtabs: function (tab_name) {
+    // Return subtabs for a given tab.
+    var subtabs_by_tab = {
+      lobbyists: {
+        default_subtab: 'lobbyists',
+        lobbyists: {
+          row_component: LobbyistRow,
+          endpoint: 'lobbyists/json/lobbyists',
+          default_key: 'law_project_count',
+          default_order: -1,
+          keys: [
+            {
+              key: 'slug',
+              title: 'Pavadinimas',
+              explanation: undefined,
+              icon: undefined,
+              order: 1},
+            {
+              key: 'law_project_count',
+              title: 'Įtakoti įstatmai',
+              explanation: 'Skaičiuojamas bendras kiekis įtakoktų teiės aktų.',
+              icon: 'users icon', order: -1},
+            {
+              key: 'client_count',
+              title: 'Užsakovai',
+              explanation: undefined,
+              icon: '', order: -1},
+            {
+              key: 'avg_passed_law_project_ratio',
+              title: 'Įtakojimo sėkmė',
+              explanation: 'Skaičiuojama, kokia dalis iš visų įtakotų teisės aktų projektų buvo priimti.',
+              icon: '', order: -1}
+          ]
+        },
+        suggester: {
+          row_component: SuggesterRow,
+          endpoint: 'lobbyists/json/lobbyists',
+          default_key: 'law_projec_count',
+          default_order: 1,
+          keys: [
+            {
+              key: 'slug',
+              title: 'Pavadinimas',
+              explanation: undefined,
+              icon: undefined,
+              order: 1},
+            {
+              key: 'law_project_count',
+              title: 'Įtakoti įstatmai',
+              explanation: 'Skaičiuojamas bendras kiekis įtakoktų teiės aktų.',
+              icon: 'users icon', order: -1},
+            {
+              key: 'number_of_suggestions',
+              title: 'Teikta pastabų',
+              explanation: 'Skaičiuojamas bendras kiekis teiktų pastabų visiems teisės aktams.',
+              icon: '', order: -1},
+            {
+              key: 'avg_passed_law_project_ratio',
+              title: 'Įtakojimo sėkmė',
+              explanation: 'Skaičiuojama, kokia dalis iš visų teiktų pastabų buvo priimti.',
+              icon: '', order: -1}
+          ]
+        }
+      }
+    };
+    return subtabs_by_tab[tab_name];
+  },
+
+  getSubtab: function (tab, subtab) {
+    // Return a subtab subtab_name for tab tab_name.
+    var self = this;
+    subtabs = self.getSubtabs(tab);
+    subtab = (subtab.startsWith('suggester') ? 'suggester' : subtab)
```

Comment:
Because astrange way to reuse the same component for `suggest_state` and `suggester_other`.